lib/runner/cli/cli.js

```diff
@@ -177,6 +178,20 @@ class CliRunner {
     this.createTestRunner();
     this.setupConcurrency();
 
+    analyticsCollector.updateSettings(this.test_settings);
+    analyticsCollector.updateLogger(Logger);
+
+    analyticsCollector.event('nw_test_run', {
+      arg_env: this.argv.env,
```

Comment:
This isn't relevant because it could be anything defined by users. The only relevant thing would be to say if it's 'default' or not.

Reply:
removed for now, will add if we need it later for information regarding parallel execution